core/src/retransmit_stage.rs

```diff
@@ -58,6 +58,9 @@ const DEFAULT_LRU_SIZE: usize = 10_000;
 // it doesn't pull up too much work.
 const MAX_PACKET_BATCH_SIZE: usize = 100;
 
+const CLUSTER_NODES_CACHE_CAP: usize = 8; // Number of epochs.
+const CLUSTER_NODES_CACHE_TTL: Duration = Duration::from_secs(5);
```

Comment:
All callers are using the same values, move these consts into ClusterNodesCache::new()?

Reply:
You could do something like ClusterNodesCache::new(Option<(cap, ttl)>) and use these as defaults if no values are provided. At least it would be explicit that the values can be chosen by the caller. That said, I'm not opposed to leaving it as written.